internal/compiler/tests/syntax/elements/popup.slint

```diff
@@ -3,17 +3,17 @@
 
 
 export component Bar {
```

Comment:
please leave this component as is to test the old property, and add another component that use the new property.  also ass a ^warning for testing the warning.

Reply:
I see that you added another file (instead of another comment, but this is fine)
But you should also revert the changes in this file.

... And also add a test for the warning